### DISCUSSION TITLE
Keys: Back to December (Taylor Swift)

### DISCUSSION THREAD

**Discussion by JmDemisana** [Monika-After-Story/MonikaModDev#7796] [general - piano notes]
I did create the JSON file, but I don't know how expressions work so I just copied a snip from the Your Reality JSON. hehe

#6512 - Download Here

_The Keys:_
Intro:
W 9 Y E R R E Y
W 9 Y E R R E Y
Verse:
I'm so glad you made time to see me
2 U Y 6 6 6 6 6 Y
How's life, Tell me how's your fam'ly
U Y 6 6 6 6 6 Y
I haven't seen you in a while
U Y U Y U Y U W 9 W 9 Y
We`ve been good, busier than ever
2 U Y 6 6 6 6 6 Y
We small talk, work and the weather
U Y 6 6 6 6 6 Y
Your guard is up and I know why
U Y U Y U Y U W 9 W 9 Y
Bridge:
Because the last time you saw me
E R Y 9 U U Y
Is still burned in the back of your mind
Y Y Y Y Y Y E W
You gave me roses and I left them there to
die
E E R 9 U Y 6 6 6 R Y
Chorus:
So this is me swallowing my pride
9 U 9 9 U 9 9 9
Standing in front of you saying I'm sorry for
that night
9 U 9 9 U 9 9 U 9 9 9 E Y 6 Y Y
And I go back to December all the time
Y U Y Y U 9 9 9 6 6 Y
It turns out freedom ain't nothing but
missing you
9 U 9 9 U 9 9 9 9 U 9
Wishing I'd realized what I had when you
were mine
9 U 9 9 U 9 9 9 E Y 6 Y Y
I'd go back to December, turn around and
make it all right
Y U Y Y U 9 9 9 U 9 U 9 U Y R
I go back to December all the time
Y U 9 9 9 6 6 Y
Interlude:
W 9 Y E R R E Y
W 9 Y E R R E Y
Verse:
These days I haven't been sleeping
2 U Y 6 6 6 6 6 Y
Staying up playing back myself leaving
U Y 6 6 6 6 6 Y
When your birthday passed and I didn't call
U Y U Y U Y U W 9 W 9 Y
And I think about summer, all the beautiful
times
Y Y 9 9 9 9 W 9 Y Y Y U
I watched you laughing from the passenger
side
Y Y 9 9 9 9 W 9 Y Y Y U
And realized I'd loved you in the fall
Y Y W W 9 Y Y Y U W 9 W 9 Y
Bridge:
And then the cold came, the dark days when
fear crept into my mind
E R Y 9 U U Y Y Y Y Y Y Y E W
You gave me all your love and all I gave you
was goodbye
E E R 9 U Y 6 6 6 R Y
Chorus:
So this is me swallowing my pride
9 U 9 9 U 9 9 9
Standing in front of you saying I'm sorry for
that night
9 U 9 9 U 9 9 U 9 9 9 E Y 6 Y Y
And I go back to December all the time
Y U Y Y U 9 9 9 6 6 Y
It turns out freedom ain't nothing but
missing you
9 U 9 9 U 9 9 9 9 U 9
Wishing I'd realized what I had when you
were mine
9 U 9 9 U 9 9 9 E Y 6 Y Y
I'd go back to December, turn around and
make it all right
Y U Y Y U 9 9 9 U 9 U 9 U Y R
I go back to December all the time
Y U 9 9 9 6 6 Y
Interlude:
9 9 9 E R 6 Y
9 9 9 E R 6 Y
Verse:
I miss your tan skin, your sweet smile, so
good to me, so right
Y Y E U Y E U Y E U Y Y U
And how you held me in your arms that
September night
W 9 Y E W 9 Y Y Y U 9
The first time you ever saw me cry
9 U Y E U Y 6 Y
Maybe this is wishful thinking
E Y Y Y Y R 6 Y
Probably mindless dreaming
E Y Y Y Y R 6 Y
But if we loved again I swear I'd love you
right
Y Y W 9 Y Y Y Y E E
I'd go back in time and change it but I can't
R 6 Y 6 Y 6 Y 6 E R R
So if the chain is on your door, I understand
R 6 Y 6 Y 6 Y 6 E R R
Chorus:
So this is me swallowing my pride
9 U 9 9 U 9 9 9
Standing in front of you saying I'm sorry for
that night
9 U 9 9 U 9 9 U 9 9 9 E Y 6 Y Y
And I go back to December
Y U Y Y U 9 9 9 6 6 Y
It turns out freedom ain't nothing but
missing you
9 U 9 9 U 9 9 9 9 U 9
Wishing I'd realized what I had when you
were mine
9 U 9 9 U 9 9 9 E Y 6 Y Y
I'd go back to December, turn around and
make it all right
Y U Y Y U 9 9 9 U 9 U 9 U Y R
I'd go back to December, turn around and
change my own mind
Y U Y Y U 9 9 9 U 9 U 9 U Y R
I go back to December all the time
Y U 9 9 9 6 6 Y
Outro:
W 9 Y E R R E Y
W 9 Y E R R E Y

#3765

**Comment by ghost** [comment]
thats hella long ngl

**Comment by AsuraShun** [comment]
I'm starting to think I should take piano lessons

**Comment by trinhvannam** [comment]
AsuraShun right about that :v:
